### PR TITLE
Us 11 merchant item create

### PR DESCRIPTION
**Pull request by BarryA**
## Type of change
- [x] New feature
- [ ] Bug Fix
- [ ] Test
- [ ] Refactor/Style

## Description
Relevant User Story(s): User Story 11

*Brief summary of the change(s) included in this pull request*

## Implements/Fixes Issue(s):
closes #

## Neccesary checkmarks:
- [x] All Tests are Passing
- [x] The code will run locally

## Any Outstanding Issues/Problems?
- [ ] My code is wonderful, no issues here!
- [x] Yeah, see additional words below
*Description of problem(s)*
Check last expect on Merchant's Index Spec for US-11, is it possible to put that within a within block?

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
   # /merchants/:merchant_id/dashboard
   resources :merchants, only: [] do
     resources :dashboard, module: "merchant", only: [:index]
-    resources :items, module: "merchant", only: [:index, :show, :edit, :update]
+    resources :items, module: "merchant"
```
Comment (kohljd) [inline config/routes.rb:10]: Technically, we have everything but a destroy now
`resources :items, module: "merchant", except [:destroy]`

File: spec/features/merchants/items/new_spec.rb
Comment (kohljd) [inline spec/features/merchants/items/new_spec.rb:1]: Would look at deleting the save_and_open_page and any comments you don't want to end up on the main branch

Comment (kohljd) [inline spec/features/merchants/items/new_spec.rb:1]: Would look at deleting the save_and_open_page and any comments you don't want to end up on the main branch

**Review by kohljd — CHANGES_REQUESTED** [review]
Overall good work, see comments for details on the couple requested changes.

File: spec/features/merchants/items/new_spec.rb
Comment (BarryA) [reply spec/features/merchants/items/new_spec.rb:1]: Deleted the unnecessary  save_and_open_page

File: app/controllers/merchant/items_controller.rb [hunk 2]
```diff
@@ -8,6 +8,22 @@ def show
...
   end
 
+  def new
+    @merchant = Merchant.find(params[:merchant_id])
+  end
+
+  def create
+    @merchant = Merchant.find(params[:merchant_id])
+    item = Item.new(item_params)
+    item.status = 1
```
[the remark this answers is not in the document]
Comment (BarryA) [reply app/controllers/merchant/items_controller.rb]: Did the switcharoo!

**Review by kohljd — APPROVED** [review]
Thanks for making those changes! Looks great

**Pull request merged**